test/Marketplace.test.js

```diff
@@ -293,14 +293,6 @@ describe("Marketplace", function () {
       ).to.be.revertedWith("Slot is not free")
     })
 
-    it("is rejected when request is finished", async function () {
-      await waitUntilStarted(marketplace, request, proof, token)
-      await waitUntilFinished(marketplace, slot.request)
-      await expect(
-        marketplace.fillSlot(slot.request, slot.index, proof)
-      ).to.be.revertedWith("Slot is not free")
-    })
-
```

Comment:
Btw. why are you removing this test? IMHO it should stay here no? That is valid test-case...

Reply:
wtf. That shouldn't have happened. I've removed another one that I added earlier.
Will update this.

Reply:
Undid this change. Sorry, this was a mistake.